builder.json

```diff
@@ -0,0 +1,93 @@
+                ["/opt/python/cp35-cp35m/bin/python", "-m", "pip", "install", ".", "--verbose"],
+                ["/opt/python/cp35-cp35m/bin/python", "-m", "pip", "install", "boto3", "autopep8"],
+                ["/opt/python/cp35-cp35m/bin/python", "-m", "unittest", "discover", "--verbose"],
+                ["echo", "------ Python 3.4 ------"],
+                ["/opt/python/cp34-cp34m/bin/python", "-m", "pip", "install", "--upgrade", "pip", "setuptools"],
+                ["/opt/python/cp34-cp34m/bin/python", "-m", "pip", "install", ".", "--verbose"],
+                ["/opt/python/cp34-cp34m/bin/python", "-m", "pip", "install", "boto3", "autopep8"],
+                ["/opt/python/cp34-cp34m/bin/python", "-m", "unittest", "discover", "--verbose"],
+                ["echo", "------ Python 2.7 narrow-unicode ------"],
+                ["/opt/python/cp27-cp27m/bin/python", "-m", "pip", "install", "--upgrade", "pip", "setuptools"],
+                ["/opt/python/cp27-cp27m/bin/python", "-m", "pip", "install", ".", "--verbose"],
+                ["/opt/python/cp27-cp27m/bin/python", "-m", "pip", "install", "boto3", "autopep8"],
+                ["/opt/python/cp27-cp27m/bin/python", "-m", "unittest", "discover", "--verbose"],
+                ["echo", "------ Python 2.7 wide-unicode ------"],
+                ["/opt/python/cp27-cp27mu/bin/python", "-m", "pip", "install", "--upgrade", "pip", "setuptools"],
+                ["/opt/python/cp27-cp27mu/bin/python", "-m", "pip", "install", ".", "--verbose"],
+                ["/opt/python/cp27-cp27mu/bin/python", "-m", "pip", "install", "boto3", "autopep8"],
+                ["/opt/python/cp27-cp27mu/bin/python", "-m", "unittest", "discover", "--verbose"]
+            ],
+            "run_tests": false,
+            "_comment": "manylinux has all its own build steps, turn off 'tests' which is where normal build steps are declared. using data to program sucks"
+        }
+    },
+    "targets": {
+        "android": {
+            "enabled": false,
+            "_comment": "disabled until we have a reason to support python on android"
+        }
+    },
+    "compilers": {
+        "_comment": "don't need compiler variants because awsiot is pure python. It will use precompiled awscrt wheels pulled from pip.",
+        "clang": { "enabled": false },
+        "gcc": {
+            "versions": {
+                "_comment": "run gcc 4.8 because that's what the linux wheels were made with",
+                "5": { "enabled": false },
+                "6": { "enabled": false },
+                "7": { "enabled": false },
+                "8": { "enabled": false }
+            }
+        },
+        "msvc": {
+            "versions": {
+                "_comment": "run msvc 2015 because that's what the windows wheels were made with",
+                "2017": { "enabled": false }
+            }
+        }
+    },
+    "upstream": [],
```

Comment:
Should this include the CRT?

Reply:
It just uses the published awscrt from pypi.
It doesn't build awscrt live from github